src/Accounts/Accounts/Context/GetAzureRMContext.cs

```diff
@@ -63,13 +64,32 @@ protected override IAzureContext DefaultContext
         [Parameter(Mandatory =true, ParameterSetName = ListAllParameterSet, HelpMessage ="List all available contexts in the current session.")]
         public SwitchParameter ListAvailable { get; set; }
 
+        [Parameter(Mandatory = false, ParameterSetName = ListAllParameterSet, HelpMessage = "Refresh contexts from token cache")]
+        public SwitchParameter RefreshContextFromTokenCache { get; set; }
```

Comment:
I just want to confirm that eventually we shall get rid of this parameter. Is that correct?